### DISCUSSION TITLE
[FeatureRequest] Allow same key

### DISCUSSION THREAD

**Discussion by bestpika** [wiresock/WireSockUI#34] [Ideas]
Some service-generated WireGuard configuration files may have contents similar to the following:
```ini
[Peer]
...
AllowedIPs = 0.0.0.0/0
AllowedIPs = ::/0
...
```
If you import the file at this time, you will receive the error message shown in the following image.
![image](https://github.com/wiresock/WireSockUI/assets/5698157/324c77ee-56ab-4f7a-b6e4-0cc3415a1a76)
> An item with the same key has already been added.

I know that as long as I change the content to the following, it will be fine.
```ini
...
AllowedIPs = 0.0.0.0/0, ::/0
...
```
But this will not be a problem in the official WireGuard client, so I hope to allow configuration files with this type of content and parse them correctly.
Thank you.

**Comment by bestpika** [comment]
Is this actually a problem with WireSock?

**Comment by wiresock** [comment]
WireSock and WireSockUI use distinct APIs to parse configuration files, making it vital to confirm that both can handle multiple identical keys. However, I'm uncertain if this is a top priority right now.
